applications/HDF5Application/python_scripts/core/controllers.py

```diff
@@ -15,17 +15,18 @@
 
 # STL imports
 import abc
+import typing
```

Comment:
```suggestion
from typing import Optional
```

I would only import what you need

Reply:
I usually `import` the whole module because you can use the module name as a "namespace" and avoid ambiguity, but I don't have a strong opinion here.